client/tests/conftest.py

```diff
@@ -10,7 +10,7 @@
 SUPERADMIN_ORGANIZATION = os.getenv("SUPERADMIN_ORGANIZATION", "superadmin_organization")
 SUPERADMIN_TOKEN = requests.post(
     urljoin(API_URL, "login/creds"),
-    data={"username": SUPERADMIN_LOGIN, "password": SUPERADMIN_PWD, "organization": SUPERADMIN_ORGANIZATION},
+    data={"username": SUPERADMIN_LOGIN, "password": SUPERADMIN_PWD},
```

Comment:
I don't follow this modification?

Reply:
I thought we need this to create the token but when we create the token the function is searching the user in the db and can take the organization_id from it